package.json

```diff
@@ -19,6 +19,13 @@
     "clean": "rimraf -rf ./dist *.tgz",
     "compile": "tsc -p tsconfig.json",
     "compile:build": "tsc -p tsconfig-build.json",
+    "dev:clean": "rimraf -rf ./logs",
+    "dev:list": "pm2 list",
+    "dev:stop": "pm2 stop ecosystem.config.js",
+    "dev:delete": "pm2 delete all",
+    "dev:eth-node": "hardhat node --config src/test/hardhat.config.ts",
```

Comment:
I placed the hardhat config inside **test/** because it was causing a problem with **airnode-protocol** contract compilation in the postinstall script, but this can be moved to the root folder once we get the new Airnode packages  released.

Reply:
hmm it seems to be required, otherwise it doesn't run the network